crm_request/views/request_stage.xml

```diff
@@ -1,10 +1,25 @@
 <?xml version="1.0"?>
 <odoo>
-    <record id="request_stage_claim_type_form" model="ir.ui.view">
+
+    <record id="crm_stage_claim_category_form" model="ir.ui.view">
+        <field name="name">crm.claim.stage.type.form</field>
+        <field name="model">crm.claim.stage</field>
+        <field name="inherit_id" ref="crm_claim.crm_claim_stage_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='sequence']" position="after">
+                <newline/>
+                <field name="case_default" />
+                <field name="claim_category"
+                     attrs="{'required':[('case_default','=',False)],
+                     'invisible':[('case_default','=',True)]}"/>
```

Comment:
It doesn't work. The view has duplicate fields and this field here doesn't appear.